install.sh

```diff
@@ -186,6 +186,7 @@ semverLT() {
 
 # Returns the tag of the latest stable release (in terms of semver and not of release date)
 get_latest_version() {
+    need_cmd sed
```

Comment:
Though it should be fine here, I'm always wary of using `sed` for a multi-platform purpose because it behaves slightly differently on OSX.